examples/trl_example/stack_llama_2/README.md

```diff
@@ -0,0 +1,36 @@
+# DPO pipeline for the creation of StackLlaMa 2: a Stack exchange llama-v2-7b model
```

Comment:
can we have folder structure like
example/trl/stack_llama_2 ? try to be close to the original trl git

Reply:
I just use the script under https://github.com/huggingface/trl/tree/main/examples/research_projects/stack_llama_2/scripts. what kind of folder structure are you suggesting, I will follow it.

Reply:
I would suggest to use example/trl/stack_llama_2